### DISCUSSION TITLE
404  Not Found [IP: 185.125.190.52 80]

### DISCUSSION THREAD

**Discussion by eshmakar** [cboxdoerfer/fsearch#442] [General]
Ign:14 http://ppa.launchpad.net/christian-boxdoerfer/fsearch-stable/ubuntu lunar InRelease
Err:15 http://ppa.launchpad.net/christian-boxdoerfer/fsearch-daily/ubuntu lunar Release
  404  Not Found [IP: 185.125.190.52 80]

it is not installed on debian 11

**Comment by mgw2013** [comment]
(Debian is not Ubuntu and) ppa's don't work in Debian...
